releasenotes/notes/dns-round-robin.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: feature
+area: networking
+issue: [31064]
```

Comment:
Not sure if [] works for issues. @brian-avery can you please confirm?
```suggestion
issue: 31064
```

Reply:
I had that and the test failed telling me to use a list

Reply:
Ok, so it should be.
```
issue:
  - 31064
```

Reply:
that is equivalent to what I have currently